build/scripts/.dappnode_profile

```diff
@@ -11,6 +11,8 @@ export DAPPNODE_CORE_DIR="${DAPPNODE_DIR}/DNCORE"
 
 #!ISOBUILD Do not modify, variables above imported for ISO build
 DNCORE_YMLS=$(find $DAPPNODE_CORE_DIR -name "docker-compose-*.yml" -printf "-f %p ")
+DNCORE_YMLS_ARRAY=($(find $DAPPNODE_CORE_DIR -name "docker-compose-*.yml"))
+
 alias dappnode_status='docker-compose $DNCORE_YMLS ps'
 alias dappnode_stop='docker-compose $DNCORE_YMLS stop && docker stop $(docker container ls -a -q -f name=DAppNode*)'
 alias dappnode_start='docker-compose $DNCORE_YMLS up -d && docker start $(docker container ls -a -q -f name=DAppNode*)'
```

Comment:
All these other commands would break then?

Reply:
Also, I think we can be sure that all composes will have the same version since we control that. The only scenario we don't is on an existing DAppNode if the migration script fails for some reason. But in that case it not a big deal users should never use this commands in normal situations

Reply:
They should not, actually the array is not necessary in the uninstall script.

The point is, when will be composes with different versions ?

I guess the answer is, just before the dappmanager starts and does the migration to the safe compose

Reply:
Then do you think it's worth it for all commands to use the same consistent style of one compose per call?

Reply:
no, just at the beginning since it will be the only time when composes may have "unsafe" composes